torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4661,6 +4678,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    ref=np.bitwise_not,
                    dtypes=integral_types_and(torch.bool),
                    supports_autograd=False),
+    OpInfo('bitwise_left_shift',
```

Comment:
This is really cool. We'll also need to add a few tests to test_binary_ufuncs.py:

- compare bitwise_left_shift and bitwise_right_shift tensor x tensor, tensor x scalar, and scalar x tensor variants against NumPy's left_shift and right_shift as references
- a sanity check that operator left shift works for tensor x tensor, tensor x scalar, and scalar x tensor, too